mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundWallet.tsx

```diff
@@ -18,54 +21,119 @@ export function PlaygroundWallet (): JSX.Element | null {
+            dark={tailwind('text-mono-dark-v2-700')}
+            light={tailwind('text-mono-light-v2-700')}
+            iconType='Feather'
+            name='file-plus'
+            size={18}
+          />
+        )
+      },
+      testID: 'playground_wallet_random'
+    }
+  ]
+
   return (
     <View>
       <PlaygroundTitle
-        status={{ online: wallets.length > 0, offline: wallets.length === 0 }}
-        title='Wallet'
-      />
-
-      <PlaygroundAction
-        onPress={clearWallets}
-        testID='playground_wallet_clear'
-        title='Clear stored mnemonic seed'
-      />
-
-      <PlaygroundAction
-        onPress={async () => {
-          await updateNetwork(network)
-          const data = await MnemonicUnprotected.toData([
-            'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'art'
-          ], network)
-          await setWallet(data)
-        }}
-        testID='playground_wallet_abandon'
-        title='Setup an unprotected wallet with abandon x23 + art as the 24 word'
-      />
-
-      <PlaygroundAction
-        onPress={async () => {
-          await updateNetwork(network)
-          const data = await MnemonicEncrypted.toData([
-            'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'abandon', 'art'
-          ], network, '000000')
-          await setWallet(data)
-        }}
-        testID='playground_wallet_abandon_encrypted'
-        title='Setup an encrypted wallet with abandon x23 + art as the 24 word with 000000 passcode'
-      />
-
-      <PlaygroundAction
-        onPress={async () => {
-          await updateNetwork(network)
-          const words = MnemonicUnprotected.generateWords()
-          const encrypted = await MnemonicEncrypted.toData(words, network, '000000')
-          await setWallet(encrypted)
-          await MnemonicStorage.set(words, '000000')
-        }}
-        testID='playground_wallet_random'
-        title='Setup an encrypted wallet with a random seed using 000000 passcode'
+        status={{ online: wallets.length > 0, offline: wallets.length === 0, type: PlaygroundStatusType.secondary }}
+        title='WALLET'
       />
+      <ThemedViewV2
+        dark={tailwind('bg-mono-dark-v2-00')}
+        light={tailwind('bg-mono-light-v2-00')}
+        style={tailwind('rounded-lg-v2 px-5')}
+      >
+        {
+        dataLists.map((dataList, index) => (
+          <PlaygroundAction
+            key={index}
+            // eslint-disable-next-line react/jsx-handler-names
+            onPress={dataList.onPress} // let it be just for dfi testing. it still working
```

Comment:
can remove this comment? 🤔 and also what does it mean?

Reply:
done cleanup